### PR TITLE
fix: get rid snail_case

### PR DESCRIPTION
**Pull request by ivyoverflow**
#10 is closed.

### CODE REVIEW

File: book/internal/handler/book_test.go [hunk 1]
```diff
@@ -17,9 +17,9 @@ import (
 	"github.com/ivyoverflow/pub-sub/book/internal/handler"
 	"github.com/ivyoverflow/pub-sub/book/internal/lib/types"
 	"github.com/ivyoverflow/pub-sub/book/internal/model"
-	mock_repository "github.com/ivyoverflow/pub-sub/book/internal/repository/mock"
+	mockrepository "github.com/ivyoverflow/pub-sub/book/internal/repository/mock"
```
Comment (kiryalovik) [inline book/internal/handler/book_test.go:20]: You can use some abbreviation like `repomock`, `svcmock` etc.

File: book/internal/handler/book_test.go [hunk 2]
```diff
@@ -28,9 +28,9 @@ func TestBookHandler_Insert(t *testing.T) {
 		name                    string
 		inputString             string
 		expectedString          string
-		mockBehaviorIDGenerator func(*mock_service.MockIDGeneratorI)
+		mockBehaviorIDGenerator func(*mockservice.MockIDGeneratorI)
```
Comment (kiryalovik) [inline book/internal/handler/book_test.go:31]: When you name some structure inside the package you don't need to duplicate package naming: 
`svcmock.IDGenerator`. And why there's `I` at the end of the name. Is it an interface in the mock package?

**Review by kiryalovik — CHANGES_REQUESTED** [review]

File: book/internal/handler/book_test.go (hunk 2 above)
Comment (ivyoverflow) [reply book/internal/handler/book_test.go]: Well, the mocks are generated automatically and I cannot change the mock.go file.

File: book/internal/handler/book_test.go (hunk 1 above)
Comment (ivyoverflow) [reply book/internal/handler/book_test.go]: Yeah, thanks, changes [here](https://github.com/ivyoverflow/pub-sub/pull/21/commits/1641b14edd69bdcb61b717778ada3f80b55bf784).

File: book/internal/handler/book_test.go (hunk 2 above)
Comment (kiryalovik) [reply book/internal/handler/book_test.go]: https://golang.org/doc/effective_go.html#interface-names

Comment (kiryalovik) [reply book/internal/handler/book_test.go]: -mock_names: A list of custom names for generated mocks. This is specified as a comma-separated list of elements of the form Repository=MockSensorRepository,Endpoint=MockSensorEndpoint, where Repository is the interface name and MockSensorRepository is the desired mock name (mock factory method and mock recorder will be named after the mock). If one of the interfaces has no custom name specified, then default naming convention will be used

https://github.com/golang/mock

Comment (ivyoverflow) [reply book/internal/handler/book_test.go]: I've uploaded the solution.

**Pull request merged**